docker/etc/my_init.d/10_dokku_init

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+set -eo pipefail
+
+log-info() {
+  declare desc="Log info formatter"
+  echo "    $*" 1>&2
+}
+
+log-fail() {
+  declare desc="Log fail formatter"
+  echo "!   $*" 1>&2
+  exit 1
+}
+
+main() {
+  if [[ ! -d /mount ]]; then
+    log-info "Creating missing /mount"
+    mkdir -p /mount
+  fi
+
+  directories=("/etc/ssh" "/home/dokku" "/var/lib/dokku/config" "/var/lib/dokku/data")
```

Comment:
putting my functional programmer hat on, i think i'd prefer to see these sections broken up into their own functions with descriptive names. i think this helps in the self document code aspect of maintainability but will defer to you on whether you think it's worth it